web/modules/weather_data/src/Service/WeatherDataService.php

```diff
@@ -62,13 +69,34 @@ public function __construct(ClientInterface $httpClient, TranslationInterface $t
 
     $this->currentConditions = FALSE;
 
+    $this->apiCache = [];
+
     $this->legacyMapping = json_decode(
       file_get_contents(
         __DIR__ . "/legacyMapping.json"
       )
     );
   }
 
+  /**
+   * Get data from the weather API.
+   *
+   * The results for any given URL are cached for the duration of the current
+   * response. The cache is not persisted across responses.
+   *
+   * Disable phpcs on the next line because it does not like method names with
+   * sequential uppercase characters, but... I'm not camel-casing "API".
```

Comment:
Maybe we can disable this rule globally for phpcs because I dont like camel casing acronyms either!